# /model/world/exceptions.py

class PathNotFoundException(Exception):
    """Raised when the A*-algorithm cannot find a path"""
    pass
